docs/getting-started.rst

```diff
@@ -1,11 +1,9 @@
 Getting Started with OpenTelemetry Python
 =========================================
 
-This guide will walk you through instrumenting a Python application with ``opentelemetry-python``.
+This guide walks you through instrumenting a Python application with ``opentelemetry-python``.
 
-For more elaborate examples, see `examples`.
```

Comment:
Why remove this?

Reply:
So it was a little confusing to me because it read like it should have been pointing to some link with examples. Do you know what is meant by that sentence or where it's supposed to direct people?

Reply:
Yes I believe that is where it is referring to ^

Reply:
Fixed.